Packages.props

```diff
@@ -44,8 +44,6 @@
     <PackageReference Update="CluedIn.CrawlerIntegrationTesting" Version="$(_CluedIn)" />
     <PackageReference Update="CluedIn.DataStore" Version="$(_CluedIn)" />
     <PackageReference Update="RestSharp" Version="106.10.1" />
-    <PackageReference Update="Dapper" Version="2.0.35" />
-    <PackageReference Update="Dapper.SqlBuilder" Version="2.0.35" />
```

Comment:
Where are we sourcing these from now?

Reply:
We've never used them, think they were dregs left over from when we first started